internal/pod_models.go

```diff
@@ -527,21 +527,20 @@ func configureSidecarContainer(container *corev1.Container, initMode bool, proce
 	if hasTrustedCAs {
 		sidecarArgs = append(sidecarArgs, "--copy-file", "ca.pem")
 	}
+
+	if optionalCluster == nil {
+		sidecarArgs = append(sidecarArgs, "--require-not-empty")
+		sidecarArgs = append(sidecarArgs, "fdb.cluster")
+	}
+
 	if optionalCluster != nil {
```

Comment:
nit: I think an `else` might be slightly more readable here

Reply:
I would keep it as it is. This method needs some refactoring anyways to reduce the different statements (and will eventually go away once the split image is removed).

Reply:
Created https://github.com/FoundationDB/fdb-kubernetes-operator/issues/2179 for it.